docs/DeveloperGuide.md

```diff
@@ -142,17 +156,51 @@ _Written by: Author name_
 
 ### Add Command
 
-{Describe the implementation for the Add Command} 
 
-_Written by: Author name_
+**This feature allows the local and external (handled by Storage class) storage of transaction entries by the user.**
+
+The `AddCommand` inherits properties from the abstract `Command` class.he inheritance of `Command` from `AddCommand` is
+shown below.
+
+<p align="center">
+    <img src="images/AddCommandClassDiagram.png">
+    <br />
+    <i>Figure 3.1: Class Diagram for AddCommand Showing Inheritance of Command</i>
+</p>
+
+`AddCommand` is dependent on `CommandParser` which accesses its `COMMANDWORD` and creates a new `AddCommand` object. 
+It is also associated with `TransactionList`, `Ui`, `Storage`, which are used in `AddCommand#execute()`; and the `Duke` 
+which calls for `AddCommand#execute()`. Lastly it is also associated with `ParameterParser` which calls for 
+`AddCommand#getMandatoryTags()`. The relationship between the classes are shown below.
+
+<p align="center">
```

Comment:
Good that there are class diagrams shown for the command. Note that we are encouraged to provide a sequence diagram and steps on how to perform the Add feature ([sample link](https://se-education.org/addressbook-level3/DeveloperGuide.html#proposed-undoredo-feature) ignore the change of states diagram)